src/azure-cli/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -420,3 +434,21 @@ def load_command_table(self, _):
         g.custom_command('delete', 'server_key_delete', confirmation=True)
         g.custom_show_command('show', 'server_key_get')
         g.command('list', 'list')
+
+    with self.command_group('mysql server ad-admin',
+                            mysql_adadmin_sdk,
+                            client_factory=cf_mysql_server_ad_administrators_operations,
+                            is_preview=True) as g:
+        g.custom_command('create', 'server_ad_admin_set', supports_no_wait=True)
```

Comment:
Because we support --no-wait for create command, we should provide another `wait command` to let users wait until  condition matched.